### ISSUE TITLE
Test the wasm version of libzim/kiwix-lib with more complex return types

### ISSUE THREAD

**Issue by mossroy** [kiwix/kiwix-js#515]
This is a follow-up of #116.

We'll have to retrieve byte arrays (for images, videos etc).
Embind should support that : https://emscripten.org/docs/porting/connecting_cpp_and_javascript/embind.html , but it needs to be tested

**Comment by kelson42** [comment]
Do we have more answers on this meanwhile?

**Comment by mossroy** [comment]
No, we have other issues with emscripten/kiwix-lib before tackling this
